benchmark_utils/generate_config/generate_config_per_dataset.py

```diff
@@ -47,6 +50,9 @@
 
         for filepath in filenames:
             name = filepath.stem  # Remove the .py suffix
+            if name.startswith('deep'):
```

Comment:
Need to also add:
`if not name.endswith('.py')`

This will ensure with os files like `.DS_STORE`